tests/acceptance/tests/Settings/BasicConsentBannerAcceptAllCookies.spec.ts

```diff
@@ -4,11 +4,16 @@ test('As a shop customer, I want to use an "Accept All Cookies" button in the ba
     ShopCustomer,
     StorefrontHome,
     TestDataService,
-    DefaultSalesChannel,
+    InstanceMeta,
 }) => {
+    // eslint-disable-next-line playwright/no-conditional-in-test
+    if (InstanceMeta.isSaaS) {
+        // eslint-disable-next-line playwright/no-skipped-test
+        test.skip();
```

Comment:
Why you skip that test? In SaaS we have the same behaviour. Because we can't set system config values in SaaS?

Reply:
The test does not work on saas, because it's not possible to invalidate the http cache on demand. That's required because the view depends on the system config value we set

Reply:
I will create a ticket that we enable as soon as https://github.com/shopware/shopware/pull/5925 is merged and deployed.